content/soarian/dstu2/medications/medication-statement.md

```diff
@@ -0,0 +1,83 @@
+---
+title: MedicationStatement | Soarian DSTU 2 API
+---
+ 
+# MedicationStatement
+
+* TOC
+{:toc}
+
+## Overview
+
+For the MedicationStatement resource query, the API returns a consolidated list of medications that the provider has defined for the patient.  The list represents the most current discharge medication reconciliation (DMR) list the provider has charted in Soarian.
+
+As the most current medication list, if a patient was taking a medication prior to admission or was given a medication during his stay, but in the discharge process the physician determined these medications no longer apply, it will be marked stopped.  Soarian clients can configure a preference to only show active medications or to also show stopped medications.  
+
+To avoid patient confusion about the definitive active medication list, no in-progress medications during an active visit are returned; these medications are managed by providers.  If there is an active visit without a DMR, we will return an Informational Message indicating that Medication reconciliation has not been performed.
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_medication_statement, :dstu2) %>
+
+## Custom Extensions
+The [bundle extension] can be returned, see possible codes in the [errors section] below.
+
+  Status Extension:
+
+The URL for this status extension is defined as `Additional status`
+
+ID                                | Value\[x] Type      | Description
+----------------------------------|---------------------|----------------------------------------------------------------------------------
+`MedicationStatement/Staus`          | [`string`]       | This element describes the additional status information of the MedicationStatement. Possible values are 'New', 'Continued' and 'Modified' |
```

Comment:
Use the tick marks to make these possible values look like codes. Also, make sure the case is accurate (from the example, it looks like these should all be lowercase).

Is there a better description of this? What is the "additional status" describing? The status of reconciliation?